install_scripts/templates/kubernetes/init.sh

```diff
@@ -1275,6 +1292,9 @@ case "$STORAGE_PROVISIONER" in
 
         if isRook1; then
             MAINTAIN_ROOK_STORAGE_NODES=1
+            if [ -z "$DISABLE_ROOK_OBJECT_STORE" ]; then
```

Comment:
We have installs of Rook 1.0.0 with the bug that takes down the gateway so we should check for isRook103 here.